FilePersistence/src/version_control/merge/ListMergeComponentV2.h

```diff
@@ -49,7 +49,17 @@ class FILEPERSISTENCE_API ListMergeComponentV2 : public MergePipelineComponent
 
 	private:
 
-		using IdToIndexMap = QMultiHash<Model::NodeIdType, QPair<QString, MergeChange::Branches>>;
+		struct LabelData{
+				QString label;
+				MergeChange::Branches branch;
+		};
+		struct IdPosition{
+				Model::NodeIdType id;
+				int baseIndex;
```

Comment:
just to make sure this structure is alwasy properly initialized, please use default initialization here and the for the next two lines. In general, always default initialize primitive types and enums.

Default initialization means to add the `{}`, as in `type name{};`